packages/wallets/solflare/src/metamask/wallet.ts

```diff
@@ -55,16 +49,15 @@ export class SolflareMetaMaskWallet implements Wallet {
     }
 
     get chains() {
-        return ['solana:mainnet', 'solana:devnet', 'solana:testnet', 'solana:localnet'] as const;
+        return SOLANA_CHAINS.slice();
```

Comment:
Just use the existing chains. However, if Solflare's MM Snap doesn't actually support all these, we should only use the ones that work.